index.js

```diff
@@ -191,3 +191,17 @@ projectButton.forEach((el) => el.addEventListener('click', () => {
     closeTag.onclick = closeIt;
   }
 }));
+
+const email = document.getElementById('email');
+const form = document.getElementById('form');
+const errorMsg = document.getElementById('error');
+
+form.addEventListener('submit', (e) => {
+  const emailValue = email.value.trim();
+  const lowerCase = emailValue.toLowerCase();
+
+  if (!emailValue.match(lowerCase)) {
+    e.preventDefault();
+    errorMsg.innerHTML = 'You entered an invalid email. please make it all lowercase';
+  }
+});
```

Comment:
Hello @team, good so far validating the form 💪🏻.

- [ ] [REQUIRED] Your form is well validated but for some reason when I enter an email with all caps letters, on submitting the event does not trigger an error message. I suggest that you look into this issue and make the necessary changes 😄.